homeassistant/components/demo/alarm_control_panel.py

```diff
@@ -42,6 +43,11 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
                         CONF_DELAY_TIME: datetime.timedelta(seconds=0),
                         CONF_TRIGGER_TIME: datetime.timedelta(seconds=10),
                     },
+                    STATE_ALARM_ARMED_VACATION: {
+                        CONF_ARMING_TIME: datetime.timedelta(seconds=5),
+                        CONF_DELAY_TIME: datetime.timedelta(seconds=0),
+                        CONF_TRIGGER_TIME: datetime.timedelta(seconds=10),
+                    },
```

Comment:
Why do we change the demo integration in this PR?

Reply:
For tests to pass. Without this tests do not pass

Reply:
Ah, the demo integration is using the manual alarm_control_panel, I didn't see that 👍